icinga2-ansible-no-ui/tasks/icinga2_Ubuntu_install.yml

```diff
@@ -2,15 +2,15 @@
 - name: install urllib3
   pip:
     name: urllib3
-    version: 1.7.1
+    version: 1.22
```

Comment:
Have you tested this recently?  Yesterday on one of the iterations of the icinga machines this was causing ansible failures because 1.22 could not be found (only 1.21 and 1.23)